### PR TITLE
Create db-repositories-test-container workflow

### PR DESCRIPTION
**Pull request by phiz71**
## Issue

https://gravitee.atlassian.net/browse/APIM-2635

## Description

create db-repositories-test-container workflow

### CODE REVIEW

File: .circleci/ci/src/jobs/job-jdbc-test-container.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const notifyOnFailureCmd = NotifyOnFailureCommand.get(dynamicConfig);
+    dynamicConfig.addReusableCommand(restoreMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(saveMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(notifyOnFailureCmd);
+
+    return new reusable.ParameterizedJob(
+      JdbcTestContainerJob.jobName,
+      UbuntuExecutor.create('medium', true),
+      new parameters.CustomParametersList([
+        new parameters.CustomParameter('version', 'string', '', 'type and version of the database to test'),
```
Comment (jourdiw) [inline .circleci/ci/src/jobs/job-jdbc-test-container.ts]: What about renaming this to `jdbcType`?

File: .circleci/ci/src/jobs/job-mongo-test-container.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    const saveMavenJobCacheCmd = SaveMavenJobCacheCommand.get();
+    const notifyOnFailureCmd = NotifyOnFailureCommand.get(dynamicConfig);
+    dynamicConfig.addReusableCommand(restoreMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(saveMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(notifyOnFailureCmd);
+
+    return new reusable.ParameterizedJob(
+      MongoTestContainerJob.jobName,
+      UbuntuExecutor.create(),
+      new parameters.CustomParametersList([new parameters.CustomParameter('version', 'string', '', 'version of the database to test')]),
```
Comment (jourdiw) [inline .circleci/ci/src/jobs/job-mongo-test-container.ts]: What about naming this to `mongoVersion`?

File: .circleci/ci/src/jobs/job-redis-test-container.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    const saveMavenJobCacheCmd = SaveMavenJobCacheCommand.get();
+    const notifyOnFailureCmd = NotifyOnFailureCommand.get(dynamicConfig);
+    dynamicConfig.addReusableCommand(restoreMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(saveMavenJobCacheCmd);
+    dynamicConfig.addReusableCommand(notifyOnFailureCmd);
+
+    return new reusable.ParameterizedJob(
+      RedisTestContainerJob.jobName,
+      UbuntuExecutor.create(),
+      new parameters.CustomParametersList([new parameters.CustomParameter('version', 'string', '', 'version of Redis to test')]),
```
Comment (jourdiw) [inline .circleci/ci/src/jobs/job-redis-test-container.ts]: Same thing here

**Review by Okhelifi — APPROVED** [review]
👍🏼 nice

**Review by jourdiw — APPROVED** [review]
LGTM

**Pull request merged**